modules/cnf-du-management-pods.adoc

```diff
@@ -0,0 +1,356 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-management-pods.adoc_{context}"]
+
+= Management pods
```

Comment:
I love the brevity of the title, but consider adding more.  Maybe:
* Cluster management pods
* Pods for cluster management workloads
* Cluster management workloads
* Cluster management workloads and pods